Kernel/Devices/FUSEDevice.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2024, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Devices/DeviceManagement.h>
+#include <Kernel/Devices/FUSEDevice.h>
+#include <Kernel/FileSystem/FUSE/Definitions.h>
+#include <Kernel/FileSystem/FUSE/FUSEConnection.h>
+
+namespace Kernel {
+
+UNMAP_AFTER_INIT NonnullLockRefPtr<FUSEDevice> FUSEDevice::must_create()
+{
+    // FIXME: Find a way to propagate errors
```

Comment:
If we initialize this device during the boot init sequence, maybe this FIXME is not really useful...